ci/tests/general/general_vdisk.py

```diff
@@ -162,8 +163,13 @@ def write_to_volume(vdisk=None, vpool=None, location=None, count=1024, bs='1M',
 
         if input_type not in ('null', 'zero', 'random'):
             raise ValueError('Invalid input type provided')
-        if not root_client.file_exists(location):
-            raise ValueError('File {0} does not exist on Storage Router {1}'.format(location, root_client.ip))
+        command = """import os, json
```

Comment:
This is very complex
Please use our Remote extension for things like this
Perhaps put it in a separate method similar to execute_command_on_node

import os
from ovs.extensions.generic.remote import Remote
is_link = False
with Remote('127.0.0.1', [os]) as remote:
    is_link = remote.os.path.islink('/var/log/ovs/lib.log')